sdks/python/apache_beam/io/datastore/v1/helper.py

```diff
@@ -82,3 +94,148 @@ def str_compare(s1, s2):
     return -1
   else:
     return 1
+
+
+def get_datastore(project):
+  """Returns a Cloud Datastore client."""
+
+  credentials = googledatastore.helper.get_credentials_from_env()
+  datastore = Datastore(project, credentials)
+  return datastore
+
+
+def make_request(project, namespace, query):
+  """Make a Cloud Datastore request for the given query."""
+
+  req = datastore_pb2.RunQueryRequest()
+  req.partition_id.CopyFrom(make_partition(project, namespace))
+
+  req.query.CopyFrom(query)
+  return req
+
+
+def make_partition(project, namespace):
+  """Make a PartitionId for the given project and namespace."""
+
+  partition = entity_pb2.PartitionId()
+  partition.project_id = project
+  if namespace is not None:
+    partition.namespace_id = namespace
+
+  return partition
+
+
+def retry_on_rpc_error(exception):
+  """A retry filter for Cloud Datastore RPCErrors."""
+
+  if isinstance(exception, RPCError):
+    if exception.code >= 500:
```

Comment:
Isn't this same as the default (retry_on_server_error) filter ? Should we use that instead ?